back/boxtribute_server/app.py

```diff
@@ -65,11 +78,15 @@ def before_sentry_send(event, hint):
     configure_app(
         app,
         *blueprints,
-        host=os.environ["MYSQL_HOST"],
-        port=int(os.environ["MYSQL_PORT"]),
+        # always used
         user=os.environ["MYSQL_USER"],
         password=os.environ["MYSQL_PASSWORD"],
         database=os.environ["MYSQL_DB"],
+        # used for connecting to development / CI testing DB
+        host=os.getenv("MYSQL_HOST"),
+        port=int(os.getenv("MYSQL_PORT", 0)),
+        # used for connecting to Google Cloud from GAE
         unix_socket=os.getenv("MYSQL_SOCKET"),
+        replica_socket=os.getenv("MYSQL_REPLICA_SOCKET"),
```

Comment:
We need to remember to set this variable in the contexts in circleci

Reply:
I already set it as a project-wide variable